mithril-common/src/signable_builder/cardano_transactions.rs

```diff
@@ -222,6 +232,34 @@ mod tests {
         }
     }
 
+    #[tokio::test]
+    async fn test_compute_merkle_root_order_of_block_range_does_not_matter() {
```

Comment:
Is this test relevant?

Reply:
The goal of this test is to make more explicit the fact that the order in which the block ranges appears doesn't matter.
This is a fact that we discovered when we adapted the previous tests, we naively multiplied the block number of each transactions by 10 and by doing so the test was red (because the transactions were now in different block ranges and the order of the block ranges doesn't matter here).